ee_tests/src/launcher-import.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { $, browser } from 'protractor';
+
+import * as logger from './support/logging';
+import { screenshotManager } from './support/screenshot_manager';
+import { CreateApplicationPage } from './page_objects/launcher.page';
+import { LoginInteractionsFactory } from './interactions/login_interactions';
+import { LauncherInteractionsFactory } from './interactions/launcher_interactions';
+import { newSpaceName } from './support/space_name';
+import { Header } from './page_objects/app/header';
+
+describe('Launcher tests for import flow', () => {
+         let spaceName = newSpaceName(),
+         header = new Header($('f8-app'));
+
+        let createApplicationPage = new CreateApplicationPage();
+
+         beforeAll( async () => {
+          logger.info('-------before All-------');
+          browser.ignoreSynchronization = true;
+          browser.driver.manage().window().setSize(1920, 1080);
+          let loginInteractions = LoginInteractionsFactory.create();
+          await loginInteractions.login();
+          await header.recentItemsDropdown.selectCreateSpace();
```

Comment:
Please do not access menus and page objects directly from spec, use interactions for that. You can create new method in AccountHomeInteractions called createSpaceWithExistingCodebase.

Reply:
I'll change it. Thanks